documentation/features/scheduling.rst

```diff
@@ -60,30 +62,63 @@ The full list of flex-context fields is as follows:
      - ``{"sensor": 6}``
        or
        ``"0.12 EUR/kWh"``
-     - The price of producing energy. Can be (a sensor recording) market prices, but also CO₂ intensity - whatever fits your optimization problem. (This field replaced the ``production-price-sensor`` field. [#old_sensor_field]_)
+     - The price of producing energy.
+       Can be (a sensor recording) market prices, but also CO₂ intensity - whatever fits your optimization problem, as long as the unit matches the ``consumption-price`` unit. (This field replaced the ``production-price-sensor`` field. [#old_sensor_field]_)
    * - ``site-power-capacity``
-     - ``"45kW"``
+     - ``"45kVA"``
      - Maximum achievable power at the grid connection point, in either direction [#asymmetric]_ (defaults to the Asset attribute ``capacity_in_mw``).
+       Becomes a hard constraint in the optimization problem, which is especially suitable for physical limitations.
    * - ``site-consumption-capacity``
      - ``"45kW"``
-     - Maximum consumption power at the grid connection point [#consumption]_ (defaults to the Asset attribute ``consumption_capacity_in_mw``). If ``site-power-capacity`` is defined, the minimum between the ``site-power-capacity`` and ``site-consumption-capacity`` will be used.
+     - Maximum consumption power at the grid connection point (defaults to the Asset attribute ``consumption_capacity_in_mw``).
+       If ``site-power-capacity`` is defined, the minimum between the ``site-power-capacity`` and ``site-consumption-capacity`` will be used. [#consumption]_
+       If a ``site-consumption-breach-price`` is defined, the ``site-consumption-capacity`` becomes a soft constraint in the optimization problem.
+       Otherwise, it becomes a hard constraint.
+
+   * - ``site-consumption-breach-price``
+     - ``"1000 EUR/kW"``
+     - The price of breaching the ``site-consumption-capacity``.
+       Can be (a sensor recording) contractual penalties, but also a theoretical penalty just to allow the scheduler to breach the consumption capacity, while influencing how badly breaches should be avoided.
```

Comment:
The PR description mentions that " if set, site-consumption-capacity becomes a soft constraint". That seems to me to be a pretty relevant effect and worth mentioning here. Same for `site-production-breach-price`.

Reply:
I now see you talk about this in the `site-production-capacity` section. Maybe that section should come before these more specialized new ones, then (and still mention what they are for as people never read everything)

Reply:
I think the order is already as you suggested:

- First, the `site-consumption-capacity` explains that it becomes a soft constraint in case the `site-consumption-breach-price` is set.
- Then, the `site-consumption-breach-price` is explained directly after.

Likewise for production.